src/common/snippets/src/pass/mha_tokenization.cpp

```diff
@@ -260,24 +264,20 @@ ov::snippets::pass::TokenizeMHASnippets::TokenizeMHASnippets(const SnippetsToken
         if (!is_matmul0_supported(matmul0))
             return false;
 
-        const auto matmul0_prc = op::Brgemm::get_output_type(matmul0->get_input_element_type(0), matmul0->get_input_element_type(1));
-        // Between MatMul0 and Softmax will be the one Loop because of LoopFusing optimization.
-        // The Loop will have one Buffer with the same shape both on input and output.
-        // Need to check for precision to get if we need one more register for Buffer
-        if (matmul0_prc.size() != ov::element::f32.size()) {
-            if (buffer_count < 2)
-                buffer_count++;
-        }
-
```

Comment:
Did we remove this block because we are going to support only fp32 in dynamism for now?

Reply:
Because it's outdated code part. See: we check for `buffer_count < 2` however before we set `buffer_count = 2` - ??? - It seems like this lines are never executed
Moreover, as I wrote above, after MatMul0 we always have one Buffer -> so I set `uniqie_buffer_reg_group_count = 1` by default.
No another reason